Article/Article.js

```diff
@@ -88,6 +88,44 @@ const data = [
   }
 ];
 
+const showArticles = document.querySelector('.articles'); 
+
+data.forEach(article =>{
+  showArticles.appendChild(createArticle(article.title, article.date, article.firstParagraph, article.secondParagraph, article.thirdParagraph)); 
```

Comment:
When you have lines that run off the page like this, you can put each argument on a new line:
`showArticles.appendChild(createArticle(
    article.title,
    article.date,
    ....
`